components/raftstore/src/store/msg.rs

```diff
@@ -308,6 +308,25 @@ where
         context: GetEntriesContext,
         res: Box<RaftlogFetchResult>,
```

Comment:
question: Will the compiler be confused which RaftlogFetchResult to use? As it's imported from store::RaftlogFetchResult and defined below too.

Reply:
my bad, define it twice when resolving the conflict.